src/main/java/ca/corefacility/bioinformatics/irida/config/web/IridaUIWebConfig.java

```diff
@@ -52,7 +52,6 @@
 @Import({ WebEmailConfig.class, IridaApiSecurityConfig.class })
 public class IridaUIWebConfig implements WebMvcConfigurer, ApplicationContextAware {
 	private static final String SPRING_PROFILE_PRODUCTION = "prod";
-	private static final String SPRING_PROFILE_DEVELOPMENT = "dev";
```

Comment:
Removed unused variable, the rest of changes in file are just the formatting changes applied.